src/groovy/au/org/emii/gogoduck/job/Job.groovy

```diff
@@ -37,6 +37,10 @@ class Job {
         new URL("$serverUrl/aggr/$uuid")
     }
 
+    URL getStatusUrl() {
+        new URL("$serverUrl/job/$uuid")
```

Comment:
I'm not sure whether this is the best place for this method though.  `Job` shouldn't really "know" anything about how to construct URLs.  It's kind of the inverse of what's in `UrlMappings`, maybe in there?